openstack/blockstorage/v2/extensions/volumeactions/requests.go

```diff
@@ -0,0 +1,176 @@
+package volumeactions
+
+import (
+	"github.com/rackspace/gophercloud"
+)
+
+type AttachOptsBuilder interface {
```

Comment:
Each exported type and function in this file needs an accompanying docblock, like [this](https://github.com/rackspace/gophercloud/blob/master/openstack/blockstorage/v1/volumes/requests.go#L10-L11).